src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingPredefinedInvocationReasons.cs

```diff
@@ -6,24 +6,21 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
 {
     internal static class UnitTestingPredefinedInvocationReasons
     {
-        public const string SolutionRemoved = nameof(SolutionRemoved);
-
-        public const string ProjectParseOptionsChanged = nameof(ProjectParseOptionsChanged);
+        public const string SemanticChanged = nameof(SemanticChanged);
+        public const string Reanalyze = nameof(Reanalyze);
         public const string ProjectConfigurationChanged = nameof(ProjectConfigurationChanged);
 
-        public const string DocumentAdded = nameof(DocumentAdded);
-        public const string DocumentRemoved = nameof(DocumentRemoved);
+        public const string HighPriority = nameof(HighPriority);
 
 #if false // Not used in unit testing crawling
```

Comment:
Ah we probably need to preserve the consts (and mark them Obsolete) since these are being used in our tests http://index/?rightProject=Microsoft.CodeAnalysis.UnitTesting.SourceBasedTestDiscovery.UnitTests&file=Analyzer%5cEventProcessorInitializationTests.cs&line=21

We can remove them after I've switched over to the new EA interfaces on our side.

Reply:
note: this is the new API which you can't be referencing yet :)

Reply:
Ah ok :)